pages/folder/[folderId].tsx

```diff
@@ -62,46 +70,35 @@ export default function FolderPage() {
   const handleFolderButtonClick = (id: number, name: string) => {
     setCurrentId(id);
     setFolderName(name);
+    if (id === 0) return router.push('/folder');
+    router.push(`/folder/${id}`)
```

Comment:
> 문제 상황 3번: - /folder에서 /folder/[folderId] 페이지로 넘어갈 때, 버튼을 클릭하면 해당 폴더의 id 값으로 이동하도록 구현했는데, 처음으로 전체 폴더가 아닌 다른 폴더 버튼을 클릭했을 때, 클릭한 폴더 id 값이 14면 /folder/14로 이동해야 하는데 꼭 처음 페이지 이동이 될 때에만 /folder/0 으로 이동합니다. 두 번째, 세 번째 등등 이후 클릭에서는 폴더 id에 맞게 잘 이동하기에 어디에 문제가 있는 것인지 잘 모르겠습니다.

구현해주신 웹페이지에서 폴더 추가가 안되어서 제가 직접 확인이 불가능한데요,
코드를 봤을 때 해당 로직에서 id값으로 넘겨받는 인자가 잘 들어오는지 콘솔 찍어서 확인해보셔야 할 것 같아요.
문제가 발생하는 코드는 여기로 보입니다.

```suggestion
    console.log('## debug', name, id)

    if (id === 0) return router.push('/folder');
    router.push(`/folder/${id}`)
```